docs/admin-guide/loading.md

```diff
@@ -86,6 +86,32 @@ human_genome_build: 37
 
 ```
 
+#### Adding custom images to a case
```

Comment:
You should document types and if field is mandatory or not in load-config.md. Something like:

- **custom_images**
  -**section_one**
    - **title** _String_ Title of the image
     - **description** _String_ Replacement string if image is not shown
      - **width** _Int_ Size of image in pixels. Defaults to ...

Reply:
Sounds good